src/d3a/d3a_core/sim_results/area_market_stock_stats.py

```diff
@@ -0,0 +1,68 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+
+class StockStats:
```

Comment:
I find the name StockStats very misleading, I would prefer a name like `OfferBidTradeGraphStats`, even though it is verbose, it displays much better where this class is used.

Reply:
Please also rename all variables that contain `stock_stats`, e.g. `area_market_stock_stats `. Thanks!